packages/spacecat-shared-aa-api-client/README.md

```diff
@@ -0,0 +1,104 @@
+# Spacecat Shared - ADobe Analytics API Client
+
+A JavaScript client for Adobe Analytics API, part of the SpaceCat Shared library.
+
+## Installation
+
+Install the package using npm:
+
+```bash
+npm install @adobe/spacecat-shared-aa-api-client
+```
+
+## Usage
+
+### Creating and instance from Helix UniversalContext
```

Comment:
We should add the AAClient methods here (the ones bellow belong to the RUMAPIClient)